skyportal/models.py

```diff
@@ -17,6 +17,7 @@
 from sqlalchemy_utils import URLType, EmailType
 from sendgrid import SendGridAPIClient
 from sendgrid.helpers.mail import Mail
+from twilio.rest import Client
```

Comment:
```suggestion
from twilio.rest import Client as TwilioClient
```
because `Client` is a pretty general, potentially ambiguous name